server/oresat-live-software-server.service

```diff
@@ -1,11 +1,16 @@
 [Unit]
-Description=OreSat Live Software Server
+Description=OreSat Live Software Web Service
 After=network.target
+Requires=oresat-live-software-rx.service
+After=oresat-live-software-rx.service
 
 [Service]
-ExecStart=/usr/local/sbin/oresat-live-software-server/oresat-live-software-server-service.sh
 User=root
 Group=root
+WorkingDirectory=/usr/local/sbin/oresat-live-software-server/
+ExecStartPre=/usr/bin/npm i
+ExecStartPre=mkdir -p public/video
```

Comment:
Directory creation has already happened via `rx.service`, which is a dependency. I don't care whether we leave it here.

Reply:
true